package.json

```diff
@@ -5,7 +5,7 @@
   "cacheVersion": "2",
   "private": false,
   "dependencies": {
-    "@dhis2/app-runtime": "^2.2.2",
+    "@dhis2/app-runtime": "2.2.2",
```

Comment:
What is the reason to use an eplicit version instead a carat version range here?